dev/NumberBox/APITests/NumberBoxTests.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Common;
+using MUXControlsTestApp.Utilities;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
+using System.Linq;
+using System.Collections.Generic;
+using MUXControlsTestApp;
+using Microsoft.UI.Xaml.Controls;
+
+#if USING_TAEF
+using WEX.TestExecution;
+using WEX.TestExecution.Markup;
+using WEX.Logging.Interop;
+#else
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.VisualStudio.TestTools.UnitTesting.Logging;
+#endif
+
+namespace Windows.UI.Xaml.Tests.MUXControls.ApiTests
+{
+    [TestClass]
+    public class NumberBoxTests
+    {
+        [TestCleanup]
+        public void TestCleanup()
+        {
+            TestUtilities.ClearVisualTreeRoot();
+        }
+
+        [TestMethod]
+        public void VerifyVisualTree()
+        {
+            using (VisualTreeVerifier visualTreeVerifier = new VisualTreeVerifier())
+            { 
+                NumberBox numberBox = null;
+                RunOnUIThread.Execute(() =>
+                {
+                    numberBox = new NumberBox();
+                    numberBox.Width = 200;
+                    numberBox.Height = 100;
+                });
+                TestUtilities.SetAsVisualTreeRoot(numberBox);
+                Verify.IsNotNull(numberBox);
```

Comment:
>numberBox [](start = 33, length = 9)

Why do we have multiple baseline files with 4,5,7,8 in the naming ? How does the mapping work - which check maps to which file ?

Reply:
There is a diff in the masters between RS5 vs other - namely CornerRadius and also a missing ContentPresenter. Is that expected ?

---
In reply to: [356890868](https://github.com/microsoft/microsoft-ui-xaml/pull/1693#discussion_r356890868) [](ancestors = 356890868)